sql/core/src/main/scala/org/apache/spark/sql/sources/commands.scala

```diff
@@ -293,9 +293,18 @@ private[sql] abstract class BaseWriterContainer(
   }
 
   private def newOutputCommitter(context: TaskAttemptContext): OutputCommitter = {
-    outputFormatClass.newInstance().getOutputCommitter(context)
+    val committerClass = context.getConfiguration.getClass(
+      "mapred.output.committer.class", null, classOf[OutputCommitter])
+
+    Option(committerClass).map { clazz =>
+      val ctor = clazz.getDeclaredConstructor(classOf[Path], classOf[TaskAttemptContext])
```

Comment:
Actually, in `InsertIntoFSBasedRelation.run`, we already set the the output path through `FileOutputFormat.setOutputPath(job, qualifiedOutputPath)`. So, the output path should be set in `context`. Seems we do not need to change `newOutputCommitter` at all, right?